packages/daimo-api/src/db/db.ts

```diff
@@ -550,3 +587,15 @@ interface DeclinedRequestRow {
   decliner: string;
   createdAt: number;
 }
+
+interface RawSessionKeyRow {
+  secret: string;
+  pubkey: string;
+  created_at: Date;
+}
+
+interface SessionKeyRow {
+  deviceSecret: string;
+  devicePubkey: Hex;
+  createdAt: number;
+}
```

Comment:
the kysely codegen does this automatically now. no sweat on this PR, can show you for future DB usage